src/woo_publications/publications/models.py

```diff
@@ -471,6 +471,33 @@ def gpp_app_url(self) -> str:
         return url_template.replace("<UUID>", str(self.uuid))
 
 
+class PublicationIdentifier(models.Model):
+    id: int  # implicitly provided by django
+    publicatie = models.ForeignKey(
+        Publication,
+        verbose_name=_("publication"),
+        help_text=_("The publication that this identifier identifies."),
+        on_delete=models.CASCADE,
+    )
+    kenmerk = models.CharField(
+        _("identifier"),
+        max_length=40,
+        help_text=_("The unique IRI that identifies the publication."),
+    )
+    bron = models.CharField(
+        _("source"),
+        max_length=40,
+        help_text=_("Determines where the identifier is defined and sourced from."),
+    )
+
+    class Meta:
+        verbose_name = _("kenmerk")
+        verbose_name_plural = _("kenmerken")
```

Comment:
I think you'll want to define a unique constraint on the `bron` + `kenmerk` fields here, when the idea is that they uniquely identify a single publication.

The reality may be different, but we can relax it if needed.